tests/test_main_forge_handlers.py

```
@@ -0,0 +1,539 @@
+    create_title,
+    create_title_confirm,
+)
+from src.bot.forge_wizard import ForgeDraft, ForgeStep
+from telegram import InlineKeyboardMarkup
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+def _run(coro):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_run`

(ANN202)